common/exchange/rates.go

```diff
@@ -36,7 +36,7 @@ func ConvertCurrencyToCelo(exchangeRates common.ExchangeRates, currencyAmount *b
 	exchangeRate, ok := exchangeRates[*feeCurrency]
 	if !ok {
 		if !ok {
-			return nil, fmt.Errorf("could not convert to native from fee currency (fee-currency=%s): %w ", feeCurrency, ErrNonWhitelistedFeeCurrency)
```

Comment:
Hi @karlb, the old error message looks correct to me.

The function is converting a fee currency to celo (native) and the failure message is:
```
could not convert to native from fee currency (fee-currency=0x4546...7756): non whitelisted fee currency address
```

Which makes perfect sense to me.

I would however question the use of a doubly nested if block checking the same condition.

Reply:
I think he just wanted to change the order so that it's "**from** XXX to **to** YYY".

But good point about the double ifs, seems like an oversight from the rebase.

Reply:
If there are strong feelings about the ordering then change it, but from an english speakers perspective both statements are equivalent.

Reply:
I compared it to the error message in `ConvertCeloToCurrency` and didn't see that "from" and "to" were swapped, so I though the error messages were accidentally the same. Since the function names had the tokens in different positions, I expected the same in the message and got blind for the to/from change.

The second `if` should be removed for sure. No idea how that happened. It should not be due to a rebase on upstream, since the file does not exist on upstream.